x-pack/plugins/apm/server/routes/alerts/register_transaction_duration_alert_type.ts

```diff
@@ -147,17 +148,25 @@ export function registerTransactionDurationAlertType({
             },
           },
           aggs: {
-            latency:
-              ruleParams.aggregationType === 'avg'
-                ? { avg: { field } }
-                : {
-                    percentiles: {
-                      field,
-                      percents: [
-                        ruleParams.aggregationType === '95th' ? 95 : 99,
-                      ],
-                    },
-                  },
+            environments: {
+              terms: {
+                field: SERVICE_ENVIRONMENT,
+                missing: ENVIRONMENT_NOT_DEFINED.value,
```

Comment:
God, I hate that we still have to handle this...

Reply:
@sqren https://github.com/elastic/kibana/issues/71239 :D